frontend/src/components/atoms/IconContainer.tsx

```diff
@@ -1,31 +1,38 @@
 import React from 'react';
-import { Box, Container } from '@chakra-ui/react';
+import { chakra, HTMLChakraProps, Image } from '@chakra-ui/react';
+import { motion, HTMLMotionProps } from 'framer-motion';
 import { SustainabilityGoal } from '../../types/ontologyTypes';
 
 type IconContainerProps = {
   sustainabilityNode: SustainabilityGoal;
   onClick: (sdg: SustainabilityGoal) => void;
 };
 
+// Makes framer motion compatible with Typescript
+type Merge<P, T> = Omit<P, keyof T> & T;
+type MotionBoxProps = Merge<HTMLChakraProps<'div'>, HTMLMotionProps<'div'>>;
+
+export const MotionBox: React.FC<MotionBoxProps> = motion(chakra.div);
```

Comment:
This might be useful other places in the application, so I would suggest having it not be exported from IconContainer.tsx, but rather somewhere else like types.